src/common/function/src/scalars/json/jq.rs

```diff
@@ -0,0 +1,174 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt::{self, Display};
+
+use common_query::error::{InvalidFuncArgsSnafu, Result, UnsupportedInputDataTypeSnafu};
+use common_query::prelude::Signature;
+use datafusion::logical_expr::Volatility;
+use datatypes::data_type::ConcreteDataType;
+use datatypes::prelude::VectorRef;
+use datatypes::scalars::ScalarVectorBuilder;
+use datatypes::vectors::{MutableVector, StringVectorBuilder};
+
+use crate::function::{Function, FunctionContext};
+
+#[derive(Clone, Debug, Default)]
```

Comment:
```suggestion
/// A function to retrieve the value in JSON by path.
#[derive(Clone, Debug, Default)]
```

Reply:
`jq` may be not a good name, sorry for misunderstanding. jq is more complex than this function. It can transform, filter the JSON.

There are some JSON functions or operators in databend

https://github.com/datafuselabs/databend/issues/11270